airbyte-webapp/src/packages/cloud/components/experiments/SignupSourceDropdown/useGetSourceDefinitions.tsx

```diff
@@ -16,10 +16,11 @@ const fetchCatalog = async (): Promise<Catalog> => {
 };
 
 export const useGetSourceDefinitions = () => {
-  return useQuery<Catalog, Error, Catalog["sources"]>("cloud_catalog", fetchCatalog, {
+  return useQuery("cloud_catalog", fetchCatalog, {
     select: (data) => {
+      const filteredConnectors = getExcludedConnectorIds();
```

Comment:
nit: rename this variable to `excludedConnectorIds`. When I read the word `filtered`, I'm never sure if it means the things that were "filtered out", or if it means what is remaining after _other_ things have been filtered out.